spec/index.bs

```diff
@@ -1338,20 +1344,112 @@ Only supported capabilities may be requested in {{GPUAdapter/requestDevice()}};
 requesting unsupported capabilities results in failure.
 
 The capabilities of a [=device=] are determined in "[=a new device=]" by starting with the adapter's
-defaults (no features and the default [=supported limits=])
+defaults (from {{adapter/[[default feature level]]}})
 and adding capabilities as requested in {{GPUAdapter/requestDevice()}}.
 These capabilities are enforced regardless of the capabilities of the [=adapter=].
 
 <p tracking-vector>
 For privacy considerations, see [[#privacy-machine-limits]].
 
+### Feature Levels ### {#feature-levels}
```

Comment:
Since we already have something called a "feature" maybe they should be called "Capability Levels" or "Capability Tiers")